docs/useful_scripts_and_tasks.adoc

```diff
@@ -40,6 +40,9 @@ v|`npm run build \-- --validate`
 v|`npm run build \-- --watch`
 | Rebuilds planner library on file changes
 
+| lint
+v|`npm run lint-check`
```

Comment:
This shouldn't be part of `build` section. All the scripts in build section start with `npm run build ...`.